ggshield/cmd/iac/scan/iac_scan_utils.py

```diff
@@ -64,7 +64,7 @@ def filter_iac_filepaths(
 def get_iac_tar(directory: Path, ref: str, exclusion_regexes: Set[Pattern]) -> bytes:
     filepaths = get_iac_filepaths(directory, ref)
 
-    def _accept_file(path: Path, content: str) -> bool:
+    def _accept_file(path: Path, open_file: Callable[[], str]) -> bool:
```

Comment:
open_file is not used in _accept_file.

Why not remove the second argument and only keep path ?
It seems the only place the fhis function is used is in tar_from_ref_and_filepath but this does not use the open_file function.